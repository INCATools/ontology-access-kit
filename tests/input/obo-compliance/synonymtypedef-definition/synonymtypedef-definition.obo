format-version: 1.4
synonymtypedef: S:1 "example"
ontology: synonymtypedef-definition

