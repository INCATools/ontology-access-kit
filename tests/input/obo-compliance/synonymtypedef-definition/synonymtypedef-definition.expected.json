{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/synonymtypedef-definition.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/S_1",
      "lbl" : "example"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#SynonymTypeProperty",
      "lbl" : "synonym_type_property",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}